lua/starfall/libs_sh/builtins.lua

```diff
@@ -761,6 +761,35 @@ function builtins_library.requiredir(path, loadpriority)
 	return returns
 end
 
+--- Returns an included script as a function.
+-- @param string path The file path to include. Can be absolute or relative to calling file. Make sure to --@include it
+-- @return function? Compiled function corresponding to file, or nil if not found.
+-- @return string? Error message, or nil if found
+function builtins_library.getFunction(path)
```

Comment:
Not a fan of this function name. Can't you just use `loadstring(getScript(path))` anyway?

Reply:
Maybe getScriptFunction is a better name.

Reply:
Or make getScript return two values.

Reply:
Yeah, I think getScript (returns code and function) and getScripts is a good addition.

Reply:
Should `getScripts` behave this way too? This would cause it to make two tables copies instead of just one. Should it also do this when another chip has been specified? Returning another chip's functions could potentially be unsafe.

Reply:
Ah, right I forgot it can take other chips. Probably shouldn't return functions in that case.

Reply:
Maybe it should be seperated in getScriptFunction afterall